src/database.rs

```diff
@@ -287,7 +287,7 @@ impl Database {
 
                         if record.value == crate::config::REMOVE_TOMBSTONE {
                             trace!("Database.build_keydir: loading datafile No={} Path={} NumRecords={}: Removing key", file_id, &entry.display(), counter);
-                            let _ignore = keydir.remove(&record.key).unwrap_or_default();
```

Comment:
`remove` here returns an `Error<()>`.

assigning a `()` (Unit) to a variable doesn't make much sense.